bin/varnishd/cache/cache_ban_lurker.c

```diff
@@ -149,13 +150,15 @@ ban_lurker_getfirst(struct vsl_log *vsl, struct ban *bt)
 		} else if (oc == &oc_mark_end) {
 			assert(move_oc == 0);
 
-			/* hold off to give lookup a chance and reiterate */
-			VSC_C_main->bans_lurker_contention++;
-			Lck_Unlock(&ban_mtx);
-			VSL_Flush(vsl, 0);
-			VTIM_sleep(cache_param->ban_lurker_holdoff);
-			Lck_Lock(&ban_mtx);
-
+			if (contested) {
+				/* hold off to give lookup a chance and reiterate */
```

Comment:
My comment from #2925 still holds:


We want the holdoff to happen if there were any contested ocs. Yet if one contested oc was returned and _getfirst gets called the second time, contested will be 0 and we won't hold off.
Also I would consider the intention a micro-optimization not worth the complication.

Reply:
Ok, then. I will not merge the last commit unless others protests.